common/data_format.hpp

```diff
@@ -20,23 +20,15 @@ namespace ILLIXR {
 
 using ullong = unsigned long long;
 
-// Data type that combines the IMU and camera data at a certain timestamp.
-// If there is only IMU data for a certain timestamp, img0 and img1 will be null
-// time is the current UNIX time where dataset_time is the time read from the csv
-struct imu_cam_type : public switchboard::event {
```

Comment:
#363 is a large commit that contains a lot of changes. You may need to rebase your changes on top of the current master, and make sure the `revert #363` is not included.